src/components/Button.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { TouchableHighlight, ViewPropTypes, Text, Dimensions } from 'react-native'
+import LinearGradient from 'react-native-linear-gradient'
+
+const gradientColors = ['#4444B2', '#37D0D0']
+
+const Button = (props) => {
+  return (
+    <TouchableHighlight {...props} elevation={10} style={[props.style, styles.button]}>
+      <LinearGradient start={{x: 0.0, y: 1.0}} end={{x: 1.0, y: 1.0}} style={styles.gradient} colors={gradientColors}>
+        <Text style={styles.text}>{props.title}</Text>
```

Comment:
we need to localize this text too, so I think you should be using Translate component here instead

Reply:
Agree. Done.